src/Build.UnitTests/Evaluation/Evaluator_Tests.cs

```diff
@@ -4277,18 +4277,20 @@ public void VerifyDTDProcessingIsDisabled2()
 
                 File.WriteAllText(projectFilename, projectContents);
 
-                Project project = new Project(projectFilename);
+                Project project = new(projectFilename);
 
-                MockLogger logger = new MockLogger();
+                MockLogger logger = new();
                 project.Build(logger);
             }
             finally
             {
                 Thread.Sleep(500);
+                t.IsAlive.ShouldBeTrue();
+                t.Abort();
+                Thread.Sleep(500);
```

Comment:
nit: Maybe it would be beneficial to use async `Task.Delay`. This thread wouldn't be blocked for other tests in case of parallel test run.

Reply:
I don't think it matters because we run tests within a project serially anyway, but it shouldn't hurt.